src/app/banking/vouchers/view-voucher.component.html

```diff
@@ -0,0 +1,89 @@
+<page-layout [ready]="data$ | async">
+  <page-content *ngIf="data$ | async; else loading" [heading]="data.type.name"
+    [mobileHeading]="data.type.name"
+    [headingActions]="headingActions$ | async">
+
+    <div class="d-flex flex-grow-1 flex-column">
+      <div
+        [class]="(layout.gtsm$ | async) ? 'd-flex flex-row-reverse justify-content-between' : 'd-flex flex-column' ">
```

Comment:
Same for class: you don't need a dynamic behavior: just use `class="d-flex flex-column flex-md-row-reverse justify-content-md-between"`. Watch out that we generally really switch layout on `sm`, not `md`, so you probably want them with `sm` instead.